### DISCUSSION TITLE
Tables are not populating while trying to use the seed command with comma separated --files

### DISCUSSION THREAD

**Discussion by krsharodh** [adonisjs/core#2329] [Help]
I am trying to seed multiple files at the same time in the local system as well as in the Heroku portal for hosting the application using the command:
adonis seed --files=Seeder1.js,Seeder2.js,CommonSeeder.js --force
It prompts that the database has seeded. However, there isn't any data in the 3 tables after running the command.

It gets populated if I run the seed commands separately as follows but the issue with that is one seeding command resets the data seeded by the other command:
adonis seed --files=Seeder1.js
adonis seed --files=Seeder2.js
adonis seed --files=CommonSeeder.js

Is there a workaround for seeding multiple files in one command?

**Comment by thetutlage** [comment]
When you define multiple files together, are your seed files even loading? You can check this by dropping a `console.log` statement in the file

**Reply by krsharodh** [reply]
No, it only loads the last file passed to the files attribute.

**Reply by thetutlage** [reply]
I recommend wrapping the file names inside quotes like this

```
adonis seed --files="Seeder1.js,Seeder2.js,CommonSeeder.js"
```
